specificatie/genereervariant/openapi.yaml

```diff
@@ -2632,8 +2632,8 @@ paths:
                   to a temporary overloading or maintenance of the server.
                 instance: https://datapunt.voorbeeldgemeente.nl/api/v1/resourcenaam?parameter=waarde
                 code: notAvailable
-  /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/privaatrechtelijkebeperkingen/{privaatrechtelijkebeperkingidentificatie}:
-    get:
+  ? /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/privaatrechtelijkebeperkingen/{privaatrechtelijkebeperkingidentificatie}
+  : get:
```

Comment:
... echt _elke_ keer. Waar komt dit vandaan? Deze rommel breekt mijn code generatie.

Reply:
Ik heb er wat mee geëxperimenteerd Rik. Het heeft te maken met de lengte van de endpoint (relatieve url) De resolver doet dit. Misschien kan @MelvLee hier iets in betekenen ?

Reply:
Ik heb gekeken of ik in de code van de resolver kan vinden waar het resolven gebeurt. Het is mij nog niet gelukt. Persoonlijk denk ik niet dat dit snel gefixt kan worden, ook als er in code de fout gevonden wordt. In de resolver zit namelijk ook codegeneratie functionaliteit.

Voor nu zal ik kijken of ik een scriptje kan toevoegen die na het resolven de ? en :  verwijdert.

Dan kunnen we kijken naar een structure oplossing. Op dit moment zie ik drie oplossingen:
- zoeken naar een andere resolver. Ik heb tot nu toe nog niet één gevonden die het perfect doet
- de namen van de parameters inkorten. Ipv kadastraalonroerendezaakidentificatie en privaatrechtelijkebeperkingidentificatie zou je ook kunnen kiezen voor kadastraalonroerendezaakid en privaatrechtelijkebeperkingid
- paths niet meer referencen. Dus één openapi.yaml en één of meerdere domain.yaml om de yaml bestanden beheersbaar te houden. Persoonlijk ben ik hier niet een groot voorstander van, want ook al werkt het dan in dit geval. Ik weet niet of de resolver er tegen kan als een path nog langer wordt

Reply:
Naja, er is mee te leven, moet er gewoon aan denken om dit elke keer te herstellen 😇 

Overigens is de `:` enkel verdwaald; deze hoort bij het _path_ op de vorige regel.